### PR TITLE
Fix ATDD checkstyle.

### PR DESCRIPTION
**Pull request by Settis**

### CODE REVIEW

File: services/src/atdd/src/main/java/org/openkilda/topo/TopoSlug.java [hunk 1]
```diff
@@ -31,51 +27,91 @@
...
-    public static final String toString(Switch aSwitch) {
-        return "S." + aSwitch.getId();
+    /**
+     * Returns a string representation of the switch.
+     *
+     * @param theSwitch the switch.
+     * @return string representation.
+     */
+    public static final String toString(Switch theSwitch) {
+        return "S." + theSwitch.getId();
```
Comment (surabujin) [inline services/src/atdd/src/main/java/org/openkilda/topo/TopoSlug.java]: A-Switch is our internal term. We do not have "the-switch" term.

File: services/src/atdd/src/main/java/org/openkilda/topo/MockController.java [hunk 2]
```diff
@@ -13,30 +13,28 @@
...
-/**
- * 
- */
 package org.openkilda.topo;
 
 /**
- * @author carmine
+ * The mock controller.
  *
+ * @author carmine
```
Comment (surabujin) [inline services/src/atdd/src/main/java/org/openkilda/topo/MockController.java]: this is ... extra data. Remove it.

File: services/src/atdd/src/main/java/org/openkilda/topo/TopoSlug.java (hunk 1 above)
Comment (Settis) [reply services/src/atdd/src/main/java/org/openkilda/topo/TopoSlug.java]: Well, style checker ask to change `aSwitch` name. We need to think how to change style rules then.

Comment (Settis) [reply services/src/atdd/src/main/java/org/openkilda/topo/TopoSlug.java]: The exception in stylecheck config was added.

**Pull request merged**